### DISCUSSION TITLE
https://kdheepak.com/blog/publish-documents-with-quarto/

### DISCUSSION THREAD

**Discussion by giscus** [kdheepak/kdheepak.github.io#21] [Comments]
# https://kdheepak.com/blog/publish-documents-with-quarto/

https://kdheepak.com/blog/publish-documents-with-quarto/

**Comment by weigao-123** [comment]
Hi Dheepak,

Thank you for sharing this! Recently, I am very interested in it, but I saw that you converted some of the `qmd` files into `ipynb` files, is there any other benefits of using `ipynb`?
